apps/i18n/dance/fr_fr.json

```diff
@@ -1,19 +1,12 @@
 {
-  "danceAiFeedbackAiBlockInWorkspaceWithNoParameters": "⚠️ Il y a un bloc intitulé \"I.A. génère effet.\" sans emojis choisis. Choisis trois emojis afin que l'I.A. génère l'effet.",
   "danceAiFeedbackAiBlockNoParameters": "⚠️ Choisis trois emojis pour l'I.A. pour générer un effet.",
   "danceAiFeedbackAiBlockNoParametersOnEvent": "⚠️ L'effet n'a pas changé après 2 mesures. Choisis trois emojis pour l'I.A. afin de générer l'effet.",
-  "danceAiFeedbackAiBlockNotConnectedSetup": " ⚠️ Connecte le bloc `I.A. génère effet` au bloc `configuration`",
-  "danceAiFeedbackConnectPressUpArrowEvent": "⚠️ Appuie sur la flèche orientée vers le haut ▲ après avoir appuyé sur *\"Démarrer\"*.",
   "danceAiFeedbackCustomize": "⚠️ Modifie l'effet généré par l'I.A.. Tu peux régénérer l'effet, soit personnaliser le code écrit par l'IA, ou recommencer avec de nouveaux émojis.",
-  "danceAiFeedbackEffectChangeAfterMeasures": "⚠️ Connecte le bloc `I.A. génère effet` à l'événement `à 2 mesures`.",
   "danceAiFeedbackKeyPress": "⚠️ T'as appuyé sur une touche, mais votre danseur n'a effectué aucun mouvement.",
   "danceAiFeedbackKeyPressRightOrLeft": "⚠️ Appuie sur la flèche droite ou gauche pendant la danse.",
-  "danceAiFeedbackMatchDancer": "⚠️ Tu as besoin d'avoir le même danseur sélectionné dans le bloc `Faire un nouveau danseur` et dans le bloc `Les danseurs faire`.",
   "danceAiFeedbackNeedNewDancer": "⚠️ Tu dois créer un danseur",
   "danceAiFeedbackNewMove": "⚠️ Ajoute un bloc pour que le danseur effectue un nouveau mouvement.",
-  "danceAiFeedbackNoAiBlock": "Ajouter le bloc `effet généré par l'I.A.` dans les réglages ",
   "danceAiFeedbackNoNewMove": "Votre danseur n'a effectué aucun autre mouvement. Continuez à essayer",
-  "danceAiFeedbackSequence": "Vérifiez l'ordre de vos blocs. Le bloc \"créer un nouveau danseur\" doit être placé avant le bloc \"les danseurs dansent\".",
   "danceAiModalBack": "Retour",
```

Comment:
I see a bunch of strings deleted, we should double check those are deleted on purpose and not by accident.